terraform/create.go

```diff
@@ -57,6 +62,22 @@ func (t *Terraform) Create() error {
 		return err
 	}
 
+	uploadBinary := false
+	binaryPath := ""
```

Comment:
Should we use `var` for this?

Reply:
Both are okay. We should just be consistent throughout. Have we been using var elsewhere ?

Reply:
It's just because you are setting the defaults. Otherwise the short form is what we always used throughout the repository I believe.

Reply:
done